src/aind_data_transfer/transformations/compressors.py

```diff
@@ -166,3 +166,31 @@ def scale_read_blocks(
                     "stream_name": read_block["stream_name"],
                 }
             )
+
+
+class ImagingCompressors:
+
+    class Compressors(Enum):
+        """Enum for compression algorithms a user can select"""
+
+        blosc = Blosc.codec_id
+
+    compressors = [member.value for member in Compressors]
+
+    @staticmethod
+    def get_compressor(compressor_name, kwargs):
```

Comment:
In this context, should the parameter kwargs have double pointers to include n parameters that will be passed into Blosc?

Reply:
good catch, will fix